README.md

```diff
@@ -207,6 +204,19 @@ var argentina = PostalCode.Parse("Z1230ABC");
 argentina.ToString("AR"); // Z 1230 ABC
 ```
 
+### Sex
+Represents a sex based on an ISO 5218 code.
+
+``` C#
+var sex = Sex.Female;
+var parsed = Sex.Parse("Male");
```

Comment:
You don't do anything with this, should it have an example or commented out value?